src/Dapr.Messaging/Dapr.Messaging.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <Description>This package contains the reference assemblies for developing messaging services using Dapr.</Description>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+        <PackageId>Dapr.Messaging</PackageId>
+        <Title>Dapr Messaging SDK</Title>
+        <Description>Dapr Messaging SDK for building applications that utilize messaging components.</Description>
+        <VersionSuffix>alpha</VersionSuffix>
```

Comment:
Probably reasonable to add an "alpha" suffix but otherwise keep the assembly version the same as other Dapr libraries.  (I had some feedback on the jobs PR related to this.)